internal/runners/initialize/init.go

```diff
@@ -288,31 +287,28 @@ func (r *Initialize) Run(params *RunParams) (rerr error) {
 		}
 	}
 
-	var executorsPath string
-	if !condition.RuntimeDisabled() {
-		// Solve runtime
-		solveSpinner := output.StartSpinner(r.out, locale.T("progress_solve"), constants.TerminalAnimationInterval)
-		bpm := bpModel.NewBuildPlannerModel(r.auth)
-		commit, err := bpm.FetchCommit(commitID, r.prime.Project().Owner(), r.prime.Project().Name(), nil)
-		if err != nil {
-			solveSpinner.Stop(locale.T("progress_fail"))
-			logging.Debug("Deleting remotely created project due to runtime setup error")
-			err2 := model.DeleteProject(namespace.Owner, namespace.Project, r.auth)
-			if err2 != nil {
-				multilog.Error("Error deleting remotely created project after runtime setup error: %v", errs.JoinMessage(err2))
-				return locale.WrapError(err, "err_init_refresh_delete_project", "Could not setup runtime after init, and could not delete newly created Platform project. Please delete it manually before trying again")
-			}
-			return errs.Wrap(err, "Failed to fetch build result")
+	// Solve runtime
+	solveSpinner := output.StartSpinner(r.out, locale.T("progress_solve"), constants.TerminalAnimationInterval)
```

Comment:
Again, just removed the top-level `if` conditional. Nothing changed inside the block.